src/components/i18n.js

```diff
@@ -1,16 +1,15 @@
 import i18n from "i18next";
 import Backend from "i18next-xhr-backend";
-import LanguageDetector from "i18next-browser-languagedetector";
+// import LanguageDetector from "i18next-browser-languagedetector";
 import { initReactI18next } from "react-i18next";
 
 i18n
   .use(Backend)
-  .use(LanguageDetector)
+  // .use(LanguageDetector)
```

Comment:
This week let's try to figure out how to use LanguageDetector without it causing routing issues!

Reply:
Sure!